cli.js

```diff
@@ -71,6 +71,7 @@ var cli = meow([
 	'  --watch, -w      Re-run tests when tests and source files change',
 	'  --source, -S     Pattern to match source files so tests can be re-run (Can be repeated)',
 	'  --timeout, -T    Set global timeout',
+	'  --pool-size, -z  Set worker pool size (BETA)',
```

Comment:
Perhaps we shouldn't document this option until it's solid. On the other hand that's what we tried with the watcher and without documenting it people won't find it.